doc/spec.adoc

```diff
@@ -55,6 +59,28 @@ This field can be used for layered products to enforce their dependencies so tha
 Each element is composed of:
 ** Mandatory `groupId` and `artifactId` elements that are the Maven coordinates of the required channel.
 ** Optional `version` to stick to a given channel version (instead of requiring the latest version of that channel). In the absence of this `version`, the latest version of the channel will be determined based on the Maven repository metadata.
+* A collection of `repositories` that defines repositories associated with the channel. Only listed repositories are used to resolve channel's components. Each repository is composed of:
+** A required `id` that defines the repository name used in Maven cache
+** A required `url` pointing to a default Maven repository
+* Optional `manifest` corresponding to the Channel Manifest artefact. Channel Manifest is used to define streams available in the channel.
+** One of the following, mutually exclusive fields, used to resolve the Channel Manifest:
+*** `gav` corresponds to Maven coordinates of the Manifest. It can either use format of `groupId:artifactId` (resulting in the latest available version being used) or `groupId:artifactId:version` (resulting in concrete version being used)
```

Comment:
`latest available version being used`? As determined by Maven metadata, right?

Reply:
Yes, it should use the same version resolution mechanism as artifacts (described in Component Version section)